privatek8s.tf

```diff
@@ -101,6 +101,45 @@ resource "azurerm_kubernetes_cluster_node_pool" "infracipool" {
   tags = local.default_tags
 }
 
+resource "azurerm_kubernetes_cluster_node_pool" "highmempool" {
+  name                  = "highmemlinux"
+  vm_size               = "Standard_D8s_v3"
+  os_disk_type          = "Ephemeral"
```

Comment:
Not sure of that spec for release.ci, don't know if omitting it would result in faster disks (?)